test/utilities/resetDatabaseContent.js

```diff
@@ -12,9 +12,11 @@
  */
 
 const { database } = require('../../lib/application.js');
+const { setRunsTimestamps } = require('./setRunsTimestamps.js');
 
 exports.resetDatabaseContent = async () => {
     await database.dropAllTables();
     await database.migrate();
     await database.seed();
+    await setRunsTimestamps();
 };
```

Comment:
A function named resetDatabaseContent should only reset database content to its default state, not do anything else.